src/Resources/skeleton/serializer/Normalizer.tpl.php

```diff
@@ -0,0 +1,30 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+
+class <?= $class_name ?> implements NormalizerInterface
+{
+    private $normalizer;
+
+    public function __construct(ObjectNormalizer $normalizer)
+    {
+        $this->normalizer = $normalizer;
+    }
+
+    public function normalize($object, $format = null, array $context = array()): array
+    {
+        $data = $this->normalizer->normalize($object, $format, $context);
+
+        // Here: add, edit, or delete some data
+
+        return $data;
+    }
+
+    public function supportsNormalization($data, $format = null): bool
+    {
+        return $data instanceof \App\Entity\BlogPost;
```

Comment:
@lyrixx why are you checking against `App\Entity\Blogpost`? 🤔

Reply:
Why not ? it's just an exemple, as in Voter